firefox-ios/Client/Frontend/Library/Bookmarks/Utitlity/BookmarksSaver.swift

```diff
@@ -8,7 +8,12 @@ import Shared
 
 protocol BookmarksSaver {
     /// Saves or updates a bookmark
-    func save(bookmark: FxBookmarkNode, parentFolderGUID: String) async -> Result<Void, Error>
+    func save(bookmark: FxBookmarkNode, parentFolderGUID: String) async -> Result<SaveResult, Error>
+}
+
+enum SaveResult {
```

Comment:
Just wondering instead of having an enum if we could return for as successful result always the `GUID` of the bookmark/folder even if we are updating it, so to simplify and remove the enum. Let me know what you think, thanks @MattLichtenstein

Reply:
Do you think it still makes sense to do this, even though returning the GUID from a function like `updateBookmarkNode` is just returning the same GUID that you pass in in the first place?